SyncUp.Data/AppDbContext.cs

```diff
@@ -64,6 +64,10 @@ public AppDbContext() { }
 
     public AppDbContext(DbContextOptions options) : base(options) { }
 
+    public DbSet<Group> Groups => Set<Group>();
+    public DbSet<Post> Posts => Set<Post>();
+    public DbSet<Comment> Comments => Set<Comment>();
```

Comment:
Should Events be here as well?

Reply:
![SoonCatGIF](https://github.com/user-attachments/assets/4fbdd3c5-c655-4dfd-a3cb-bc3bce0eb39b)